overview/analytics.md

```diff
@@ -21,7 +21,7 @@ There are several analytics adapter plugins available to track header bidding pe
 | PulsePoint                                                       | Contact vendor                                                                      | [Website](https://www.pulsepoint.com/header-bidding.html)        |               |
 | ShareThrough                                                     | Contact vendor                                                                      |                                                                  |               |
 | PrebidAnalytics by Roxot                                         | Paid, see [pricing](http://prebidanalytics.com/#pricing). | [Website](http://prebidanalytics.com/overview-examples)          |          0.22 |
-| PubWise                                                          | Free, see [terms](http://admin.pubwise.io/terms)                                    | [Website](https://pubwise.io/)                                   |          0.24 |
+| PubWise                                                          | Free & Paid, see [pricing](https://pubwise.io/pricing/)                                    | [Website](https://pubwise.io/pubwise/)                                   |          0.24 |
```

Comment:
nit: it would be nice if you could line up the fields here with the correct amount of whitespace.